package_scanner.nim

```diff
@@ -116,6 +117,10 @@ proc check(): int =
       echo "E: ", name, " has no license"
       result.inc()
 
+    elif pdata["url"].str.normalize.startsWith("git://github.com/"):
```

Comment:
What about BitBucket, GitLab, etc? This check shouldn't be GitHub-specific, should it?